test/cmd/dnshammer.go

```diff
@@ -121,6 +126,22 @@ func compare(a, b []string) bool {
 	return true
 }
 
+func lookupCNAMERecord(domain string) (out []string, err error) {
+	// TODO (Nino-K): Switch back to the net.LookupCNAME once this is addressed: https://github.com/golang/go/issues/50101
+	c := dns.Client{}
+	m := dns.Msg{}
+	m.SetQuestion(domain, dns.TypeCNAME)
+	r, _, err := c.Exchange(&m, "127.0.0.1:53")
+	if err != nil {
+		return nil, err
+	}
+	for _, a := range r.Answer {
+		cname := a.(*dns.CNAME)
+		out = append(out, cname.Target)
+	}
+	return out, err
```

Comment:
`return` because they're named, or (IMHO better) drop the names, declare `out` explicitly, and return `out, nil`.